pom.xml

```
@@ -28,14 +28,16 @@
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/view-job-filters-plugin</gitHubRepo>
-        <jenkins.version>2.387.1</jenkins.version>
+        <jenkins.version>2.387.3</jenkins.version>
```

revert
```suggestion
        <jenkins.version>2.387.1</jenkins.version>
```
as per https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version at least pending 2.414.1